### PR TITLE
Dynamic page

### PR DESCRIPTION
**Pull request by martinnajjar12**
### Make the page dynamic
In this milestone I've:
- given the functionality to the form I have
- implemented the remove button successfully

### CODE REVIEW

File: src/containers/BookForm.js [hunk 1]
```diff
@@ -1,19 +1,44 @@
...
 const BookForm = () => {
   const category = categories.map(category => (
     <option key={category} value={category}>{ category }</option>
   ));
+  const [inputValue, setInputValue] = useState('');
+  const [selectValue, setSelectValue] = useState('');
+
+  const handleInputChange = e => {
+    setInputValue(e.target.value);
+  };
```
Comment (wathigo) [inline src/containers/BookForm.js]: Let's make use of es6 destructuring syntax. We can write this as:
```js
 const handleInputChange = ({ target: { value } }) => {
    setInputValue(value);
  };
```

File: src/containers/BookForm.js [hunk 2]
```diff
@@ -1,19 +1,44 @@
...
+  const handleSelectChange = e => {
+    setSelectValue(e.target.value);
+  };
```
Comment (wathigo) [inline src/containers/BookForm.js]: Let's use es6 syntax

File: src/containers/BookForm.js [hunk 3]
```diff
@@ -1,19 +1,44 @@
...
         </select>
-        <input type="submit" />
+        <button
+          onClick={() => {
+            dispatch(CREATE_BOOK({
+              id: shortid.generate(),
+              title: inputValue,
+              category: selectValue,
+            }));
+          }}
```
Comment (wathigo) [inline src/containers/BookForm.js]: Implements `handleSubmit` and attaches it to form's onSubmit event ([screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleSubmit.png)).
Let's create a separate event handler method and attach it to the button onClick attribute. After dispatching the action, the state should be reset to handle another form submission.

File: src/containers/BooksList.js [hunk 4]
```diff
@@ -17,6 +17,7 @@ const BooksList = () => {
           <th>Book ID</th>
           <th>Title</th>
           <th>Category</th>
+          <th>Action</th>
         </tr>
       </thead>
       <tbody>
```
Comment (wathigo) [inline src/containers/BooksList.js:30]: **REQUIRED** 
* It implements handleBookRemove ([screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)).
* It connects REMOVE_BOOK action.
* It passes handleRemoveBook to Book component [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)

**Review by wathigo — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :o: 
Great job on this milestone.
Let's make a few changes before we proceed. See the comments :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer:
Simon Wathigo**

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi @martinnajjar12 :wave: 

Good job, I can see you have implemented the changes.
 
Move on with the next milestone :100: 

### Happy Coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Pull request merged**